src/wallet/rpcwallet.cpp

```diff
@@ -2336,6 +2336,8 @@ UniValue getwalletinfo(const JSONRPCRequest& request)
             "{\n"
             "  \"walletversion\": xxxxx,     (numeric) the wallet version\n"
             "  \"balance\": xxxxxxx,         (numeric) the total confirmed balance of the wallet in " + CURRENCY_UNIT + "\n"
+            + (fLiteMode ?
```

Comment:
Haven't dug in to the logic _too_ closely, but here you check if fLiteMode is set, and below check if it's _not_ set... is this correct?

Reply:
🙈